lib/utils/fs/createZipFile.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const globby = require('globby');
+const archiver = require('archiver');
+const BbPromise = require('bluebird');
+
+function createZipFile(srcDirPath, outputFilePath) {
+  const files = globby
```

Comment:
If we're after all files, wouldn't it be more to the point just use some recursive readdir? `globby` is more for pattern matching, so seems a bit overkill for this case

(assuming we have some util handy for that)